manager/ui/war/plugins/api-manager/html/api/api-def.html

```diff
@@ -31,20 +31,18 @@
                  apiman-i18n-key="api-def">API Definition</div>
             <div class="apiman-label-faded">
               <p apiman-i18n-key="api-def-explanation">
-                Here you can (optionally) provide definition information about your API.  This
-                information should take the form of a Swagger API definition file.  Adding a 
-                definition file will allow consumers to better understand how to use your API.
-                Simply copy/paste (or drag and drop!) your Swagger API definition into the text
-                area below.
+                It is optional to provide definition information about your API in the form of a Swagger API definition
+                file. Adding a definition file will allow consumers to better understand how to use your API. Copy/paste
+                or drag and drop your Swagger API definition into the text area below.
               </p>
             </div>
-            <div class="apiman-divider-20" ng-show="definitionType">
+            <div class="apiman-divider-20" ng-hide="definitionType === 'None'">
               <div class="alert alert-warning">
                 <span class="pficon pficon-info"></span>
                 <p apiman-i18n-key="api-def.replaceInfo">If the API is published, the endpoint in the definition
                   will be automatically replaced by the endpoint of the gateway.</p>
                 <p apiman-i18n-key="api-def.downloadInfo">You can retrieve the definition with the following link&#58;&#32;</p>
-                <p><a href="{{ apimanDefinitionUrl }}">{{ apimanDefinitionUrl }}</a></p>
+                <p><b>{{ apimanDefinitionUrl }}</b></p>

```

Comment:
`<b>` is discouraged. Either use CSS or use <strong>. There's probably an existing bootstrap or patternfly CSS attribute you can use such as `font-weight-bold`.